packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -96,33 +101,35 @@ export class RatingBase extends BaseComponent<IRatingProps, IRatingState> {
       theme: theme!
     });
 
-    for (let i = min as number; i <= (max as number); i++) {
-      const ratingStarProps: IRatingStarProps = {
-        fillPercentage: this._getFillingPercentage(i),
-        disabled: disabled ? true : false,
-        classNames: this._classNames
-      };
-
-      starIds.push(this._getStarId(i - 1));
-
-      stars.push(
-        <button
-          className={ css(this._classNames.ratingButton, {
-            [this._classNames.rootIsLarge]: size === RatingSize.Large,
-            [this._classNames.rootIsSmall]: size !== RatingSize.Large
-          }) }
-          id={ starIds[i - 1] }
-          key={ i }
-          { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
-          onFocus={ this._onFocus.bind(this, i) }
-          disabled={ disabled || readOnly ? true : false }
-          role='presentation'
-          type='button'
-        >
-          { this._getLabel(i) }
-          <RatingStar key={ i + 'rating' }  {...ratingStarProps} />
-        </button>
-      );
+    for (let i = this._min as number; i <= (max as number); i++) {
+      if (i !== 0) {
+        const ratingStarProps: IRatingStarProps = {
+          fillPercentage: this._getFillingPercentage(i),
+          disabled: disabled ? true : false,
+          classNames: this._classNames
+        };
+
+        starIds.push(this._getStarId(i - 1));
+
+        stars.push(
```

Comment:
Can you add a test case, or better, update the screener test to show a representation for the various states in rating here? (0 stars, 1 stars, etc)? We clearly didn't have coverage before and we should fix that.

Reply:
Good idea